packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -559,82 +565,46 @@ export class VerticalStackedBarChartBase extends React.Component<
     });
   };
 
-  private _onStackHover = (
-    xAxisPoint: string | number,
-    lineData: ILineDataInVerticalStackedBarChart[] | undefined,
-    mouseEvent: React.MouseEvent<SVGPathElement>,
-  ): void => {
+  private _onStackHover(stack: IVerticalStackedChartProps, mouseEvent: React.MouseEvent<SVGElement>): void {
     mouseEvent.persist();
+    this._onStackHoverFocus(stack, mouseEvent);
+  }
+
+  private _onStackHoverFocus(
+    stack: IVerticalStackedChartProps,
+    refSelected: React.MouseEvent<SVGElement> | SVGGElement,
+  ): void {
+    const lineData = stack.lineData;
     const isLinesPresent: boolean = lineData !== undefined && lineData.length > 0;
-    const found = find(
-      this._points,
-      (sinlgePoint: { xAxisPoint: string | number; chartData: IVSChartDataPoint[] }) =>
-        sinlgePoint.xAxisPoint === xAxisPoint,
-    );
     if (isLinesPresent) {
       lineData!.forEach((item: ILineDataInVerticalStackedBarChart & { shouldDrawBorderBottom?: boolean }) => {
         item.data = item.data || item.y;
         item.shouldDrawBorderBottom = true;
       });
     }
     this.setState({
-      refSelected: mouseEvent,
+      refSelected,
       isCalloutVisible: true,
       YValueHover: isLinesPresent
-        ? [...lineData!, ...found!.chartData.slice().reverse()]
-        : found!.chartData.slice().reverse(),
-      hoverXValue: xAxisPoint,
-      stackCalloutProps: found!,
-      activeXAxisDataPoint: xAxisPoint,
+        ? [...lineData!, ...stack.chartData.slice().reverse()]
```

Comment:
hey @mbest , i think while taking master to your branch , you have reverted the behavior which we merged in https://github.com/microsoft/fluentui/pull/15814/files.. we were sorting the `lineData` in descending order

could you please put it back.?

Thanks,
For making the code clean up on this.

Reply:
Merged now.

Reply:
hey, can you see line data sorted.?

i am not able to [see](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/15816/merge/charting/dist/index.html#/examples/VerticalStackedBarChart), 
![image](https://user-images.githubusercontent.com/33802398/98621931-67179a80-232e-11eb-959d-37d2efc7f5e3.png)


it should be in descending. like yellow should be on last and pink should be on first


if it's working for you in local, can you post a pic here. then i guess we are good merge this pr

Reply:
![image](https://user-images.githubusercontent.com/1230364/98657049-4b909d80-22f6-11eb-9c32-926c1e7b3200.png)